static/js/components/inputs/FieldsOfStudySelectField.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import _ from 'lodash';
+import SelectField from './SelectField';
+import FIELDS_OF_STUDY from '../../fields_of_study';
+
+let fieldOfStudyOptions = _.map(FIELDS_OF_STUDY, (name, code) => ({
+  value: code,
+  label: name
+}));
```

Comment:
oh interesting! we're doing the polyfill in `dashboard.js`, but I bet the `import` statements cause this code to be executed first. you could try moving this:

```js
import entries from 'object.entries';
if (!Object.entries) {
  entries.shim();
}
```

higher in `dashboard.js`?